src/smart-components/common/form-renderer.js

```diff
@@ -1,40 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ReactFormRender, { componentTypes } from '@data-driven-forms/react-form-renderer';
-import { layoutMapper, formFieldsMapper } from '@data-driven-forms/pf4-component-mapper';
 import Pf4SelectWrapper from '../../presentational-components/shared/pf4-select-wrapper';
+import { componentMapper, FormTemplate } from '@data-driven-forms/pf4-component-mapper';
+import FormButtons from './FormButtons';
 
-const buttonPositioning = {
-  default: {},
-  modal: {
-    buttonOrder: [ 'cancel', 'reset', 'save' ],
-    buttonClassName: 'modal-form-right-align'
-  }
-};
-
-const FormRenderer = ({ componentMapper, formContainer, ...rest }) => (
-  <div className={ buttonPositioning[formContainer].buttonClassName }>
+const FormRenderer = ({ schema, onCancel, onSubmit, initialValues }) => {
+  return (
     <ReactFormRender
-      formFieldsMapper={ {
-        ...formFieldsMapper,
-        componentMapper,
+      componentMapper={ {
+        ...componentMapper,
         [componentTypes.SELECT]: Pf4SelectWrapper
       } }
-      layoutMapper={ layoutMapper }
-      { ...buttonPositioning[formContainer] }
-      { ...rest }
+      FormTemplate={ (props) => <FormTemplate { ...props } FormButtons={ FormButtons }></FormTemplate> }
+      initialValues={ initialValues }
+      onSubmit={ onSubmit }
+      onCancel={ onCancel }
+      schema={ schema }
```

Comment:
props you are not modifying change into `{...props}`. No reason to specify them (and you won't need to define additional propTypes)